lib/presentation/notifiers/timezones_notifier.dart

```diff
@@ -7,8 +7,12 @@ class TimezoneNotifier extends StateNotifier<List<String>> {
     _init();
   }
 
-  _init() async {
-    state = await WorldTimeApi.getTimeZones() ?? [];
+  _init() {
+    print('TimezoneNotifier._init.1: $state');
+    Future.delayed(Duration(milliseconds: 1000)).then((_) async {
+      state = await WorldTimeApi.getTimeZones() ?? [];
+      print('TimezoneNotifier._init.2: $state');
+    });
```

Comment:
Why was the delay required?

Reply:
There was strange behavior then, API didn't give any data without this delay. I checked now - everything is ok, delay is removed.